chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,73 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate Quick Copy-type menulist with locales
+	 * (for locale menulists that don't update with style selection)
+	 */
+	this.populateLocaleList = function(doc, menulistID, prefLocale) {
+		if(!_initialized) this.init();
+		
+		var menulist = doc.getElementById(menulistID);
+		var popup = doc.createElement('menupopup');
+		menulist.appendChild(popup);
+		
+		var desiredLocale = "";
+		var ZoteroFallbackLocale = Zotero.locale;
+		
+		// Primary dialect conversion (e.g. "en" to "en-US")
+		if (Zotero.Styles.primaryDialects.hasOwnProperty(prefLocale)) {
+			prefLocale = Zotero.Styles.primaryDialects[prefLocale];
+		}
+		if (Zotero.Styles.primaryDialects.hasOwnProperty(ZoteroFallbackLocale)) {
+			ZoteroFallbackLocale = Zotero.Styles.primaryDialects[ZoteroFallbackLocale];
```

Comment:
Is this doing something? (Also, why capitalized?) I don't see it being used below.